src/Microsoft.DotNet.Arcade.Sdk/tools/TargetFrameworkDefaults.props

```
@@ -37,8 +37,9 @@
     <NetToolCurrent Condition="'$(DotNetBuildFromSource)' == 'true'">$(NetCurrent)</NetToolCurrent>
 
     <!-- Lowest version of .NET at the time of the release of NetCurrent that is supported by tooling.
-         Undefined when NetToolCurrent and NetToolMinimum are identical. -->
-    <NetToolMinimum>net8.0</NetToolMinimum>
+         Identical with NetCurrent when building from source. -->
+    <NetToolMinimum Condition="'$(DotNetBuildFromSource)' != 'true'">$(NetMinimum)</NetToolMinimum>
+    <NetToolMinimum Condition="'$(DotNetBuildFromSource)' == 'true'">$(NetCurrent)</NetToolMinimum>
```

We don't want NetToolMinimum to default to NetMinimum. We version those two properties differently in some cases. Basically NetTool* properties shouldn't rely on Net* properties.

```suggestion
         Identical with NetToolCurrent when building from source. -->
    <NetToolMinimum Condition="'$(DotNetBuildFromSource)' != 'true'">net8.0</NetToolMinimum>
    <NetToolMinimum Condition="'$(DotNetBuildFromSource)' == 'true'">$(NetToolCurrent)</NetToolMinimum>
```